### PR TITLE
Tsks mgmt dev

### PR DESCRIPTION
**Pull request by tol8901**
Hello, Team

Could you check my pull request, please.

Best regards,
Anatolii Kravchenko.

### CODE REVIEW

File: app/controllers/users/sessions_controller.rb [hunk 1]
```diff
@@ -1,13 +1,22 @@
 class Users::SessionsController < Devise::SessionsController
   respond_to :json
+  skip_before_action :require_no_authentication
 
-  private
 
-  def respond_with(_resource, _opts = {})
+  # def respond_with(_resource, _opts = {})
```
Comment (dmitri-wm) [inline app/controllers/users/sessions_controller.rb]: delete code, not comment

File: app/controllers/users/sessions_controller.rb [hunk 2]
```diff
@@ -1,13 +1,22 @@
 class Users::SessionsController < Devise::SessionsController
   respond_to :json
```
Comment (dmitri-wm) [inline app/controllers/users/sessions_controller.rb]: if we use API mode, do we need to specify respond_to ?

File: app/controllers/users/sessions_controller.rb (hunk 1 above)
Comment (tol8901) [reply app/controllers/users/sessions_controller.rb]: Removed in commit "[commit-29] Remove unnecessary code"

File: app/controllers/users/sessions_controller.rb (hunk 2 above)
Comment (tol8901) [reply app/controllers/users/sessions_controller.rb]: No, in API mode we don't need it.
Fixed in commit "[commit-29] Remove unnecessary code"

**Pull request merged**